src/hotspot/share/prims/jvmtiThreadState.cpp

```diff
@@ -549,10 +553,8 @@ JvmtiVTMSTransitionDisabler::VTMS_vthread_end(jobject vthread) {
   if (JvmtiExport::should_post_vthread_unmount()) {
     JvmtiExport::post_vthread_unmount(vthread);
   }
-  if (JvmtiExport::can_support_virtual_threads()) {
```

Comment:
Why is this check: if (JvmtiExport::can_support_virtual_threads()) { removed?

Reply:
The same reason as above. Thank you for checking this.